src/main/java/net/wurstclient/commands/VClipCmd.java

```diff
@@ -108,5 +109,10 @@ private void vclip(double height)
 	{
 		ClientPlayerEntity p = MC.player;
 		p.setPosition(p.getX(), p.getY() + height, p.getZ());
+		if(WurstClient.INSTANCE.getHax().jesusHack.isEnabled())
+		{
+			while(WurstClient.INSTANCE.getHax().jesusHack.isOverLiquid())
+				p.setPosition(p.getX(), p.getY() + 1, p.getZ());
+		}

```

Comment:
Agreed. Also the `vclip()` method seems like the wrong place to put a check like this. It would completely ignore existing block collision checks and change the height even if the user request an exact number (e.g. by typing `.vclip 5`).